test/app-tap/trigger.test.lua

```diff
@@ -6,12 +6,11 @@ box.cfg{
     log = "tarantool.log"
 }
 
-local trigger = require('internal.trigger')
```

Comment:
It seems to belong to one of previous commits.

Reply:
Fixed.